### PR TITLE
Create list tracker

### PR DESCRIPTION
**Pull request by jchen293**
## What?
Add functionality to create trackers in bulk for our node library.
## Why?
Currently, our node library is missing one of the core features.
## How?
Add CreateList method in Tracker class.
## Testing?
Add a unit test in tracker.js test file.

### CODE REVIEW

File: test/resources/tracker.js [hunk 1]
```diff
@@ -31,4 +31,19 @@ describe('Tracker Resource', () => {
       expect(err).to.be.an.instanceOf(NotImplementedError);
     });
   });
+
+  describe('Trackers in bulk', () => {
```
Comment (Justintime50) [inline test/resources/tracker.js:35]: Did you actually run this unit test? Our Node library is a bit different in the fact that the unit test suite makes 0 API calls and everything is mocked (we'll be fixing this shortly). I don't know that we had a way built out for the tests to accept an API key to actually make this call here.

Comment (jchen293) [reply test/resources/tracker.js:35]: Yes, I just E2E tested it.

File: src/resources/tracker.js [hunk 2]
```diff
@@ -32,5 +32,17 @@ export default api => (
     static delete() {
       return this.notImplemented('delete');
     }
+
+    static async createList(params = {}) {
+      const newParam = { trackers: params };
```
Comment (Justintime50) [inline src/resources/tracker.js:37]: `newParam` should be plural

File: src/resources/tracker.js [hunk 3]
```diff
@@ -32,5 +32,17 @@ export default api => (
...
+      try {
+        const url = 'trackers/create_list';
+        await api.post(url, { body: newParam });
+        // This endpoint does not return anything so true is return here
```
Comment (Justintime50) [inline src/resources/tracker.js:41]:
```suggestion
        // This endpoint does not return anything so true is returned here
```

**Review by Justintime50 — CHANGES_REQUESTED** [review]

**Review by Justintime50 — APPROVED** [review]

**Review by nwithan8 — APPROVED** [review]
LGTM

**Pull request merged**